codespell_lib/data/dictionary.txt

```diff
@@ -49546,6 +49551,7 @@ repostiories->repositories
 repostiory->repository
 repostories->repositories
 repostory->repository
+repot->report, repost,
```

Comment:
At least move to _rare_, `repot` is a word.

Reply:
Moved to rare.